.github/workflows/run-magento-tests.yml

```diff
@@ -5,7 +5,10 @@ on:
     branches-ignore: 
       - 'master'
       - 'rc/*/*/*'
-      
+  pull_request:
```

Comment:
Why is this needed?

Reply:
Actually, **on push**, it only works for the main Carknox repo, and the branch is not master; when I use the forked repo, so GitHub workflow works for `pritesh-lr:<other_branch>`, but When I create a Pull request, it still fails GitHub workflow, 
By adding the above code, it will work for both times **on push** if a branch is not master and **on pull request open.**